### DISCUSSION TITLE
Scaling traffic volumes / street capacity to increase performance

### DISCUSSION THREAD

**Discussion by EwoutH** [toruseo/UXsim#73] [General]
Hi @toruseo and others,

I have quite a large street network (1226 nodes and 2836 edges, after extensive simplification and pruning) that I want to simulate over extended time. I don't need it to be incredibly precise, just give a good ballpark estimate. Therefor I'm wondering if I can scale down the traffic volumes / street capacity to increase performance.

For example, could I decrease the number of vehicles by 10x and also lower the street capacity by 10x? Will that give approximately correct road speeds? Do I need to adjust other factors as well? Or are there better ways to do such a thing.

**Comment by toruseo** [comment]
I think increasing `deltan` of `World` is exactly what you want. It will give correct solution for within-link traffic congestion without altering other parameters. But please keep in mind that too large `deltan` may output strange route choice results as it aggregates a lot of travelers into one agent.
